static/app/views/monitors/components/checkInTimeline.tsx

```diff
@@ -38,35 +47,82 @@ function getBucketedCheckInsPosition(
 }
 
 export function CheckInTimeline(props: Props) {
-  const {bucketedData, start, end} = props;
+  const {bucketedData, start, end, timeWindow} = props;
+
+  function JobTickTooltip(jobTick: JobTickData) {
```

Comment:
Should we make this whole thing it's own component actually?

Reply:
Then we can pull it out into it's own component and test it

Reply:
sounds good, will pull it out and add some tests now